### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dgenr8**
Improve data directory notes and development process description.

### CODE REVIEW

**Comment by dgenr8** [comment]
@dagurval Feel free to use the edit button and commit to this branch.  We can use "Squash and merge" when ready.

File: README.md [hunk 1]
```diff
@@ -1,25 +1,28 @@
 Bitcoin XT
 ==========
 
-Bitcoin XT is an implementation of a Bitcoin full node that embraces Bitcoin's original vision of simple, reliable, low-cost transactions for everyone in the world. Bitcoin XT originated as a series of patches on top of Bitcoin Core and is now a independently maintained software fork. See our notable [features](https://bitcoinxt.software/patches.html).
+Bitcoin XT is an implementation of a Bitcoin Cash (BCH) full node that embraces Bitcoin's original vision of simple, reliable, low-cost transactions for everyone in the world. Bitcoin XT originated as a series of patches on top of Bitcoin Core and is now a independently maintained software fork. See our notable [features](https://bitcoinxt.software/patches.html).
```
Comment (dagurval) [inline README.md]: “Our notable” implies that it’s the only new features. I’d use “some selected”

File: README.md [hunk 2]
```diff
@@ -1,25 +1,28 @@
...
 
-XT uses the same data directories as Core so you can easily switch back and forth. You will keep and continue updating the same block chain.
+Bitcoin XT downloads are code signed and are [built reproducibly using gitian](https://github.com/bitcoinxt/gitian.sigs). Discussion can be found at the [XT Gitter](https://gitter.im/bitcoinxt/Lobby).
+
+Data Directory Compatibility
+============================
+
+If using a data directory from a different BCH or BTC full node, specify `-reindex` on first launch (reason: UTXO database format).
```
Comment (dagurval) [inline README.md]: Having the same utxo format won’t help because we don’t do obfuscation. It would help _from_ XT though

File: README.md [hunk 3]
```diff
@@ -1,25 +1,28 @@
...
+Bitcoin XT can run in BTC chain mode by specifying `-uahftime=0`. Please note that Bitcoin XT does not validate segwit, and the conversion notes above still apply.
+
 
 Development process
 ===================
 
-To propose a patch for inclusion or to discuss Bitcoin development in general, you are welcome to use the [mailing list](https://groups.google.com/forum/#!forum/bitcoin-xt/).
+Ideas for useful features are tracked in Issues.  Pull requests from developers, including well-done cherry picks from other clients or cryptos, are very welcome.
+
+To discuss Bitcoin Cash development in general, you are welcome to use the [mailing list](https://lists.linuxfoundation.org/pipermail/bitcoin-ml/).
```
Comment (dagurval) [inline README.md]: Remove mailing list

Comment (dgenr8) [reply README.md]: This is the new cross-implementation mailing list link.

File: README.md (hunk 2 above)
Comment (dgenr8) [reply README.md]: So, remove reason:, or update to list two reasons: you have to reindex?

File: README.md (hunk 3 above)
Comment (dagurval) [reply README.md]: My bad :-)

**Comment by dagurval** [comment]
@xt-bot retest this

**Pull request merged**